src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -394,13 +392,21 @@
     <!-- Non-null type is required for this node kind -->
     <Field Name="Type" Type="TypeSymbol" Override="true" Null="disallow"/>
 
+    <!-- Various assignable expressions, including some BoundDeconstructionVariables for nested variables -->
     <Field Name="LeftVariables" Type="ImmutableArray&lt;BoundExpression&gt;"/>
```

Comment:
Presence of these node can potentially confuse SemanticModel, etc.

---
In reply to: [65776149](https://github.com/dotnet/roslyn/pull/11715#discussion_r65776149) [](ancestors = 65776149)